backend/index.js

```diff
@@ -6,8 +6,9 @@ const router = require('./api/router')
 const app = express()
 
 app.use('/api/v1', cors(), router)
+app.use(express.static(`${__dirname}/../dist`));
 
-const port = 8001
+const port = process.env.PORT || 8001
```

Comment:
Port must also be set on `src/api.js`,

I don't know how to provide these values to the front-end, maybe we need to tweak the build.

Reply:
There is some information here: https://cli.vuejs.org/guide/mode-and-env.html#modes
It shouldn't be hard.

Reply:
I added a '.env' file with a 'DEFAULT_PORT'.
This way we will use the 'PORT' variable if it is defined by heroku. I had to prepend VUE_APP because vue only uses env variables with this prefix.

Reply:
Nevermind. This is still work in progress.